### DISCUSSION TITLE
Using Node 18 but 'npm install' is throwing a Node version error

### DISCUSSION THREAD

**Discussion by hyunwoona** [prisma/prisma#16208] [Q&A]
Hi,

I am getting the following error when I try 'npm install prisma'

```
npm ERR! code 1
npm ERR! path /Users/path/to/project/node_modules/prisma
npm ERR! command failed
npm ERR! command sh -c -- node scripts/preinstall-entry.js
npm ERR! ┌───────────────────────────────────────────┐
npm ERR! │   Prisma only supports Node.js >= 14.17   │
npm ERR! └───────────────────────────────────────────┘

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/me/.npm/_logs/2022-11-09T20_20_18_706Z-debug-0.log
```

I am using Node 18 which I installed through nvm on Mac OS.

```
$ node --version
v18.12.1
$ nvm version
v18.12.1
```

I tried using pnpm, deleting and reinstalling nvm, and removing all other node versions but I couldn't make it work.
Is this a known error? Can someone help me figure out the issue?

**Comment by janpio** [comment]
Not a universal problem:
```
C:\Users\Jan\Documents\throwaway\16208>npm install prisma

added 2 packages, and audited 3 packages in 5s

found 0 vulnerabilities

C:\Users\Jan\Documents\throwaway\16208>node -v
v18.12.1

C:\Users\Jan\Documents\throwaway\16208>npx prisma -v
prisma                  : 4.6.1
@prisma/client          : Not found
Current platform        : windows
Query Engine (Node-API) : libquery-engine 694eea289a8462c80264df36757e4fdc129b1b32 (at node_modules\@prisma\engines\query_engine-windows.dll.node)
Migration Engine        : migration-engine-cli 694eea289a8462c80264df36757e4fdc129b1b32 (at node_modules\@prisma\engines\migration-engine-windows.exe)
Introspection Engine    : introspection-core 694eea289a8462c80264df36757e4fdc129b1b32 (at node_modules\@prisma\engines\introspection-engine-windows.exe)
Format Binary           : prisma-fmt 694eea289a8462c80264df36757e4fdc129b1b32 (at node_modules\@prisma\engines\prisma-fmt-windows.exe)
Format Wasm             : @prisma/prisma-fmt-wasm 4.6.1-3.694eea289a8462c80264df36757e4fdc129b1b32
Default Engines Hash    : 694eea289a8462c80264df36757e4fdc129b1b32
Studio                  : 0.476.0

C:\Users\Jan\Documents\throwaway\16208>npm -v
8.19.2
```

Can you share the full log file?

**Reply by vladi-stevanovic** [reply]
👋 Hi @hyunwoona , did you perhaps have a chance to See Jan's recommendation? Let us know if you can share the full log file for us to continue investigating!

Or, if you've found a solution, we'd love to know how you solved this problem. Feel free to post it in this thread and "Mark as answer" if this discussion is now solved!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
